### DISCUSSION TITLE
NTSC-RT Ultimate Preset Pack v0.1.3

### DISCUSSION THREAD

**Discussion by smfthe1199** [ntsc-rs/ntsc-rs#171] [General]
### What's new?

Not much changes, the following has been implemented but not limited to:
- The removal of the older version(s) of my AV Effect, @notdigidani's camcorder preset, and @Degamisu's 80's broadcast preset, as such these are no longer available.
_Which means?_
- Replacing with the latest version of this preset (#114)
- My updated version of my effect has also been added as well (#172)

Here's the file:  
[NTSC-RT Ultimate Preset Pack V0.1.3.zip](https://github.com/user-attachments/files/16751496/NTSC-RT.Ultimate.Preset.Pack.V0.1.3.zip)

**Comment by TheSystemGuy1337** [comment]
This should be pinned.

**Comment by polikux** [comment]
You can delete this post now, I made version 0.2.0. Thanks for updating this project :D.

**Reply by smfthe1199** [reply]
I am actually going to close this post for archival purposes and put "New version available" something like that in the comments, and no problem.

**Comment by smfthe1199** [comment]
New version available: #207
